package.json

```diff
@@ -25,12 +26,15 @@
     "react-native": "https://github.com/expo/react-native/archive/sdk-37.0.0.tar.gz",
     "react-native-elements": "^1.2.7",
     "react-native-gesture-handler": "~1.6.0",
+    "react-native-ionicons": "^5.0.0-rc",
+    "react-native-paper": "^3.10.1",
```

Comment:
Same for `react-native-paper`

Reply:
We need it to use the bottom tab

Reply:
Uh, ok.
But then the documentation says we must configure babel not to include everything on production